bin/monkey-cli.ts

```diff
@@ -8,10 +8,14 @@ import { bufferToHex, generateAddress, toBuffer } from 'ethereumjs-util'
 import Ora from 'ora'
 import * as utils from 'web3-utils'
 
+import { resolve } from 'dns'
```

Comment:
Is this module installed or added to package.json?

Reply:
not used, removed